client_start_charge.go

```diff
@@ -0,0 +1,73 @@
+package monta
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+// StartChargePointsRequest is the request input to the [Client.StartChargePoints] method.
+type StartChargePointsRequest struct {
+	// The id of the team that will be paying for the charge.
+	PayingTeamID int64 `json:"payingTeamId"`
+	// Id of the charge point used for this charge.
+	ChargePointID int64 `json:"chargePointId"`
+	// If true, a charge point will be reserved and a charge object with state reserved will be returned.
+	ReserveCharge bool `json:"reserveCharge"`
+}
+
+// StartChargePointsResponse is the response output from the [Client.StartChargePoints] method.
+type StartChargePointsResponse struct {
+	// Charge that started 
+	Charge Charge `json:"charge"`
+}
+
+
+// StartChargePoints to start your charge point.
+func (c *Client) StartChargePoints(
+	ctx context.Context,
+	request *StartChargePointsRequest,
+) (_ *StartChargePointsResponse, err error) {
+	const method, path = http.MethodPost, "/v1/charges" 
+	defer func() {
+		if err != nil {
+			err = fmt.Errorf("%s %s: %w", method, path, err)
+		}
+	}()
+    var requestBody bytes.Buffer
+	if err := json.NewEncoder(&requestBody).Encode(&request); err != nil {
+		return nil, err
+	}
+
+	requestURL, err := url.Parse(apiHost + path)
+	if err != nil {
+		return nil, err
+	}
+
+	httpRequest, err := http.NewRequestWithContext(ctx, method, requestURL.String(), &requestBody) 
+	if err != nil {
+		return nil, err
+	}
+	if err := c.setAuthorization(ctx, httpRequest); err != nil {
+		return nil, err
+	}
+	httpRequest.Header.Set("content-type", "application/json")
+	httpResponse, err := c.httpClient.Do(httpRequest)
+	if err != nil {
+		return nil, err
+	}
+	defer func() {
+		_ = httpResponse.Body.Close()
+	}()
+	if httpResponse.StatusCode != http.StatusOK {
+		return nil, newStatusError(httpResponse)
+	}
+	var response StartChargePointsResponse
+	if err := json.NewDecoder(httpResponse.Body).Decode(&response); err != nil {
+		return nil, err
+	}
+	return &response, nil
+}
```

Comment:
configure your editor to add newline at the end of files

https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files

Reply:
thanks, its added now!